src/lib/division.ml

```diff
@@ -0,0 +1,57 @@
+
+(* This file is free software, part of FTW. See file "LICENSE" for more information *)
+
+(* Divisions:
+
+   These are the competitive divisions that are defined by the SCR4T *)
+
+(* Type definitions *)
+(* ************************************************************************* *)
+
+type t =
+  | Novice
+  | Intermediate
+  | Advanced
```

Comment:
Divisions are `Initié`, `Intermediaire` and `Avancé`. Shouldn't we use the french names ?